src/components/RadioButton/RadioButton.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { RadioButton } from './RadioButton';
+
+export default {
+  title: 'Components/Radio Button',
+  component: RadioButton,
+};
+
+export const _RadioButton = {
+  render: () => (
+    <>
+      <RadioButton />
+    </>
+  ),
```

Comment:
Since you are rendering only one component you don't need to wrap. 

```suggestion
  render: () => <RadioButton />
```

Reply:
Oh, sorry, that's the second time... I'll be more careful.

Reply:
No worries! :)